benches/bench.rs

```diff
@@ -62,8 +62,16 @@ fn impl_bench_parsing_debug_info<R: Reader>(
         let mut cursor = unit.entries(&abbrevs);
         while let Some((_, entry)) = cursor.next_dfs().expect("Should parse next dfs") {
             let mut attrs = entry.attrs();
-            while let Some(attr) = attrs.next().expect("Should parse entry's attribute") {
```

Comment:
Would using `while let Sone(ref attr)` or `let mut attrs = entry.attrs().by_ref()` work?

Reply:
I mentioned `while let Some(ref attr) ` in the PR text. I'll try `by_ref` later, but it seems to be for creating a reference to the iterator, not the items.

Reply:
Should have read the PR description better.

Reply:
`by_ref` made no difference.